auto_flu/analysis.py

```diff
@@ -32,7 +32,6 @@ def build_pipeline_command(config, pipeline):
             '-with-timeline', pipeline['pipeline_parameters']['timeline_path'],
     ]
     pipeline['pipeline_parameters'].pop('log_path', None)
-    pipeline['pipeline_parameters'].pop('work_dir', None)
```

Comment:
I think we can't simply skip popping the `work_dir` param here because it will then be added as `--work_dir` in the next step when we iterate over the remaining `pipeline_parameters`.

I'll merge this and follow up.

Reply:
I see it now! Thanks for catching this issue, Dan! I didn't fully grasp the requirement to remove those particular parameters from the `pipeline_parameters` dict before moving on. 

One thought I had on this was: perhaps instead of "borrowing" use of the `pipeline_parameters` dictionary for temporary loading of these parameters and then removing them, we could create a separate `nextflow_parameters` dict that tackles all the single dash Nextflow parameters: 

- `-work_dir`
- `-log`
- `-with-report`
- `-with-trace`
- `-with-timeline`

... and this dict would not be involved in the `--argument` loading / iteration. 

Not a pressing change though. Just a thought for the future.